loom/engine/bindings/loom/lmApplication.cpp

```diff
@@ -448,7 +456,7 @@ int LoomApplication::initializeCoreServices()
 
     loom_asset_subscribe(bootAssembly.c_str(), LoomApplication::__handleMainAssemblyUpdate, NULL, 0);
 
-    // And fire script executoin.
+    // And fire script execution.
```

Comment:
I'm going to have to insist on this being a breaking change as it severely impacts the user experience as we speak.